region/maridia/outer.json

```diff
@@ -6225,6 +6503,89 @@
               "usableComingIn": false,
               "openEnd": 1
             }
+          ],
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": false,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Bottom to Left)",
```

Comment:
Passing through the middle involves blind movement.  Its not that bad, the hard part would be jump morphing into the turnnel at the correct height.  And being aware that all crabs are active off camera.

Reply:
I'm on the fence about this one. It is blind, but not too challenging. The places with off screen movement currently are all harder than this, and those skipping it are all easier than this.

Reply:
I would put blind movement on `no movement items` and probably springball only too.  It can be hard to know exactly where the morph tunnel is -- its always higher than I think.  Jumping to the open tube isn't an issue.  Crabs being active off camera should be a note.